docs/id/api-reference/instruction-set/mathematics/comparison-logic.md

```
@@ -0,0 +1,9 @@
+# Perbandingan & Logika
+Instruksi-instruksi ini mengevaluasi hubungan antar nilai dan memproses ekspresi boolean untuk alur bersyarat.
+
+| Kode Operasi | Argumen | Operan (stack) | Deskripsi |
+| :--- | :--- | :--- | :--- |
+| `gt` / `lt` | tipe | val1, val2 | Lebih Dari atau Kurang Dari |
+| `ge` / `le` | tipe | val1, val2 | Lebih Dari/Kurang Dari atau Sama Dengan |
+| `eq` / `neq` | tipe | val1, val2 | Sama Dengan atau Tidak Sama Dengan |
+| `and` / `or` | - | val1, val2 | Operasi logika boolean (&& / |  | ) |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)